### DISCUSSION TITLE
Prometheus/Grafana Monitoring - no server insights

### DISCUSSION THREAD

**Discussion by Autoflow** [uyuni-project/uyuni#9278] [Q&A]
Hi everybody, I just installed monitoring with Grafana and Prometheus. For this I used the documentation from here: #https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/monitoring.html#_install_prometheus. 

Everything installed fine and now the documentation tells me: If you have also installed Grafana with the Web UI, the server insights are visible on the Uyuni Server dashboard, in the Admin -  Manager Configuration - Monitoring.

After several tries I still can't see any of the server insights. There are still no dashboards, graphs, metrics or anything else below Admin - Manager Configuration - Monitoring. Only the two buttons Enable/Disable monitoring.

Can someone please give me advice or a screenshot where to find the insights on a running system?
That would be great!

Thanks very much, Alexander

Uyuni Server Release: 2024-08-Podman
Monitoring Server: openSUSE Leap 15.6

**Comment by rjmateus** [comment]
@witekest do you have any ideas?

**Reply by witekest** [reply]
It is a documentation bug. We have never embedded or linked Grafana dashboard in Uyuni UI. The mentioned UI is meant for enabling/disabling exposition of Uyuni server metrics.
The Uyuni Server dashboard is available in Grafana UI. You have to follow [these instructions](https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/monitoring.html#_install_grafana).
